tdrs-backend/docs/github_readme_status_badges.md

```diff
@@ -55,7 +36,12 @@ The badges will report on the following criteria:
 
 ## Decision
 
-This was done to facilitate a quick view of the status of the default Github repositories. 
+This was done to facilitate a quick view of the status of the default Github repositories.
+
 ## Consequences
 
-Due to limitations imposed by Github and occasional slow sever response times from GitHub, some badges might require a page refresh to load.
\ No newline at end of file
+Due to limitations imposed by Github and occasional slow sever response times from GitHub, some badges might require a page refresh to load.
+
+## Dependabot Security Analysis
+
+Dependabot status badges have not yet been updated to work with GitHub Native Dependabot per [this open issue](https://github.com/dependabot/dependabot-core/issues/1912). In lieu of these badges links are provided to the Dependabot alerts page on raft-tech and the Security Advisories page on HHS.
```

Comment:
Just want to point this out - the badges aren't currently working correctly for GitHub Native Dependabot so they were not used in this PR.